lib/plugin-loader.js

```diff
@@ -569,63 +560,61 @@ PluginLoader.prototype = {
   _readPluginDef(pluginDescriptorFilename) {
     const pluginPtrPath = this.options.relativePathResolver(pluginDescriptorFilename,
                                                  this.options.pluginsDir);
-    bootstrapLogger.info(`Processing plugin reference ${pluginPtrPath}...`);
+    bootstrapLogger.info(`ZWED0044I`, pluginPtrPath); //bootstrapLogger.info(`Processing plugin reference ${pluginPtrPath}...`);
     if (!fs.existsSync(pluginPtrPath)) {
-      throw new Error(`${pluginPtrPath} is missing`);
+      throw new Error(`ZWED0021E - ${pluginPtrPath} is missing`);
     }
     const pluginPtrDef = jsonUtils.parseJSONWithComments(pluginPtrPath);
-    bootstrapLogger.log(bootstrapLogger.FINER, util.inspect(pluginPtrDef));
+    bootstrapLogger.debug("ZWED0121I", util.inspect(pluginPtrDef)); //bootstrapLogger.log(bootstrapLogger.FINER, util.inspect(pluginPtrDef));
     let pluginBasePath = pluginPtrDef.pluginLocation;
     if (!path.isAbsolute(pluginBasePath)) {
       pluginBasePath = this.options.relativePathResolver(pluginBasePath, process.cwd());
     }
     if (!fs.existsSync(pluginBasePath)) {
       return {location: pluginBasePath,
               identifier: pluginPtrDef.identifier,
-              error: new Error(`No plugin directory found at ${pluginPtrDef.pluginLocation}`)};
+              error: new Error(`ZWED0015E - No plugin directory found at ${pluginPtrDef.pluginLocation}`)};
      }
     let pluginDefPath = path.join(pluginBasePath, 'pluginDefinition.json');
     if (!fs.existsSync(pluginDefPath)) {
       return {location: pluginBasePath,
               identifier: pluginPtrDef.identifier,
-              error: new Error(`No pluginDefinition.json found at ${pluginBasePath}`)};
+              error: new Error(`ZWED0016E - No pluginDefinition.json found at ${pluginBasePath}`)};
     }
     let pluginDef = jsonUtils.parseJSONWithComments(pluginDefPath);
-    bootstrapLogger.log(bootstrapLogger.FINER,util.inspect(pluginDef));
+    bootstrapLogger.debug("ZWED0122I", util.inspect(pluginDef)); //bootstrapLogger.log(bootstrapLogger.FINER,util.inspect(pluginDef));
     if (pluginDef.identifier !== pluginPtrDef.identifier) {
       return {location: pluginBasePath,
               identifier: pluginPtrDef.identifier,
-              error: new Error(`Identifier doesn't match one found in pluginDefinition: ${pluginDef.identifier}`)};
+              error: new Error(`ZWED0017E - Identifier doesn't match one found in pluginDefinition: ${pluginDef.identifier}`)};
```

Comment:
Does not match what is in JSON:  "ZWED0017E":"RESERVED: %s and %s don't match - plugin ignored",

Reply:
Addressed.